tests/bindings/python/benchmark_import_results.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# Author:
+#   Dominique Blaze <contact@d0m.tech>
+
+import os
+from import_handler import oscap, result2str, get_path
+
+
+'''
+Import and browse benchmark results
+
+'''
+
+benchmark = oscap.xccdf.benchmark_import(get_path("samples/xccdf_sample_results.xml"))
+if benchmark.instance is None:
+    raise Exception("Cannot import the benchmark {0}"
+                    .format(get_path("samples/xccdf_sample_results.xml")))
+else:
+    print("Benchmark id: ", benchmark.get_id())
+    results = benchmark.get_results()
+    test_result = results.pop()
+    print(test_result.get_id())
+
+    for rs in test_result.get_rule_results():
+        print(rs.get_idref(), result2str(rs.get_result()))
+
+        if (rs.get_idref() == "R-SHOULD_PASS" and
+                rs.get_result() != oscap.xccdf.XCCDF_RESULT_PASS):
+
```

Comment:
I suggest saving the return values of `rs.get_result()` and `rs.get_idref()` to a variable.

Reply:
i will do that, but what's the interrest to use an intermediate variable ? both functions are on a different line so there is no doubt about where is the problem in case of fail ...